src/GitLabProjectHealth-Model-Importer/GLPHModelImporter.class.st

```diff
@@ -332,8 +332,11 @@ GLPHModelImporter >> importMergeResquestMerger: aGLPHEMergeRequest [
 { #category : #'import -projects' }
 GLPHModelImporter >> importProject: aProjectID [
 
-	| result projectResult |
+	| result projectResult isProjectExist |
 	('Import project with id:  ' , aProjectID printString) recordInfo.
+	
+	isProjectExist :=  (glhModel allWithType: GLHProject) detect: [ :project | project id = aProjectID ] ifNone: [ nil ].
```

Comment:
I think there is a `detect:ifOne:` or `detect:ifPresent:` method that you can use to condensate this line with the other one